packages/dev/docs/pages/releases/2025-01-15.mdx

```diff
@@ -0,0 +1,72 @@
+{/* Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+
+description: TODO
+
+date: 2025-01-15
+
+---
+
+# November 20, 2024 Release
+
+Happy New Year! We're excited to announce the first release of 2025. This release includes a number of new features, enhancements, and bug fixes. Read on for more details.
+
+## Enhancements
+
+- refactor: Remove custom hit testing in usePress - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/7427)
+- chore: Improve React Fast Refresh in storybook - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/7435)
+- RAC Breadcrumbs, ListBox, Tabs - feat: Tabs collapse behaviour - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/7202)
+Autocomplete
+  - feat: RAC Autocomplete audit for release - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/7475)
+  - chore: Remaining Autocomplete alpha release checklist audit items - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/7589)
+  - fix: Text selection with Shift + Home/End isn't working (#7228) - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/7403)
+  - fix: Autocomplete event leak and erroneous item focus after backspacing  - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/7584)
+
+## Fixes
+
+- Button
+  - fix: disable form submission for isPending - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/7498)
+- Calendar
+  - fix: Selecting an unavailable date with keyboard should not change the current selection - [@richardbann](https://github.com/richardbann) - [PR](https://github.com/adobe/react-spectrum/pull/7455)
+  - feat: Add `firstDayOfWeek` prop - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/7363)
```

Comment:
should this be under enhancements?

Reply:
Yes, I'm about to commit fixes to this page which include that.